site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
I agree on all your points. With 'future changeset' I meant to do it before pushing this PR on production :P i.e make it this week. But since it is urgent to prevent this shadowing of users / orgs let's make the important changes on this PR and leave the rest for the more distant future after we have a better sense of our needs there. I'll try to experiment with the CVGs early this week and make the changes.